packages/features/ee/teams/pages/team-members-view.tsx

```diff
@@ -56,7 +68,7 @@ const MembersView = () => {
                     ]}
                   />
                 )}
-                {team.membership.role === MembershipRole.OWNER &&
+                {/* {team.membership.role === MembershipRole.OWNER &&
                 team.membership.isMissingSeat &&
                 team.requiresUpgrade ? (
                   <Alert
```

Comment:
Do we need this for V2 teams?

Reply:
There might be teams with missing seats in V2. So yes.